src/commands/azureServiceOperators/helpers/azureservicehelper.ts

```diff
@@ -64,47 +59,35 @@ async function getOperatorsPod(
     clusterKubeConfig: string
 ): Promise<k8s.KubectlV1.ShellResult | undefined> {
     // kubectl get pods -n operators
-    const command = `get pods -n operators`;
-    const failureDescription = "Get operator pod had following error";
+    const command = `get pods -n azureserviceoperator-system`;
+    const failureDescription = "Get azureserviceoperator-system pod had following error";
 
     const result = await invokeKubectlCommand(kubectl, clusterKubeConfig, command, failureDescription);
     return result;
 }
 
-async function installOlmCrd(
+async function installCerManager(
```

Comment:
small typo - `installCertManager` (I only mention it because sometimes I search for keywords in the code to help understand how things work)